File: test/benchmark/fib.py
import time

def f(n):
    a, b = 0, 1
    for i in range(0,n):
        a, b = b, a + b
    return a

start = time.time()
answer = f(50);
elapsed = time.time() - start
print("Found answer {}".format(answer))
print("Elapsed {}".format(elapsed))

File: test/benchmark/fib.c
#include <time.h>
#include <stdio.h>

long fib(long n) 
{
  long  a = 0;
  long b = 1;
  long temp = 0;

  for (long i = 0; i < n; i++) 
  {
    temp = a+b; 
    a=b;
    b=temp;
  }
  return a;
}

int main(void) 
{
  clock_t start = clock();

  long answer = fib(50);

  clock_t end = clock();
  double elapsed = (double)(end-start) / CLOCKS_PER_SEC;

  printf("Found answer %ld\nElapsed %lf\n", answer, elapsed);

  return 0;
}

